dali/test/tensor_list_wrapper.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_TENSOR_LIST_WRAPPER_H
+#define DALI_TENSOR_LIST_WRAPPER_H
+
+namespace dali {
+namespace testing {
+
+struct TensorListWrapper {
```

Comment:
This doesn't say what it does. The name suggest that it somehow wraps TensorList (which is what we wanted) but the Run function takes just one instead of a vector of these.

Reply:
I modified Run so that it takes multiple Wrappers. I suppose this would make do?